src/applications/check-in/components/pages/nextOfKin/NextOfKinDisplay.jsx

```diff
@@ -10,7 +12,7 @@ import { URLS } from '../../../utils/navigation';
 import { EDITING_PAGE_NAMES } from '../../../utils/appConstants';
 
 export default function NextOfKinDisplay({
-  header = 'Is this your current next of kin information?',
+  header = i18next.t('is-this-your-current-next-of-kin-information'),
```

Comment:
why are some translated like this with `i18next` and others using `react-i18next`?